domain_config_ui/src/Form/SwitchForm.php

```diff
@@ -0,0 +1,113 @@
+<?php
```

Comment:
I've re-introduced the switch form for use on pages that aren't conventional config forms, such as the theme settings page. This form will reload the page when the domain is switched.